src/pjs/pjs.proto

```diff
@@ -0,0 +1,184 @@
+syntax = "proto3";
+
+package pjs;
+option go_package = "github.com/pachyderm/pachyderm/v2/src/pjs";
+
+import "google/protobuf/any.proto";
+
+message Job {
+  int64 id = 1;
+}
+
+enum JobState {
+  // UNSPECIFIED means the job state is unspecified.
+  JOB_STATE_UNSPECIFIED = 0;
```

Comment:
I don't think these enums need the type name in the value names.   The generated code calls it `JobState_JOB_STATE_UNSPECIFIED`, so it just stutters.

Reply:
I don't think you can have two enums with the same value in the same package?  Something about c++.

Reply:
That seems to be true.